builder/vmware/common/driver_player5.go

```diff
@@ -205,11 +205,28 @@ func (d *Player5Driver) Verify() error {
 	d.VmwareDriver.NetworkMapper = func() (NetworkNameMapper, error) {
 		pathNetmap := playerNetmapConfPath()
 		if _, err := os.Stat(pathNetmap); err != nil {
```

Comment:
Don't we need to change this conditional to the == nil?

Reply:
good catch.